platform-sdk/swirlds-common/src/main/java/com/swirlds/common/utility/CommonUtils.java

```diff
@@ -255,7 +255,9 @@ public static <T> T throwArgNull(final T arg, final String argName) {
      * @param arg     the argument checked
      * @param argName the name of the argument
      * @see StringUtils#isBlank(CharSequence)
+     * @deprecated use {@link com.swirlds.base.ArgumentUtils#throwArgBlank(String, String)} instead
```

Comment:
@hendrikebbers, Given these conversations and this decision, should we change the comment about what to use instead, or change the implementation of `ArgumentUtils.throwArgBlank` to be an inlined call backed by `Objects.requireNonNull`?
* [conversation 1](https://swirldslabs.slack.com/archives/C04RFBCCP8B/p1681139457284359) 
* [conversation 2](https://swirldslabs.slack.com/archives/C03DU0QGE2G/p1681242564890119)
* [Decision/Outcome](https://www.notion.so/swirldslabs/007-Java-Code-Style-a0130a7059b044f68447b9ca3a5242b3?pvs=4#fadddb0f9c974f06a8fecccff951eb96)

Reply:
Yes but we should not do that in this PR. Maybe you can create a new one?

Reply:
And maybe we deprecate `ArgumentUtils.throwArgBlank`?

Reply:
I created an issue for it: https://github.com/hashgraph/hedera-services/issues/6139

I'm good with resolving this conversation.